packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -172,6 +179,46 @@ export default class EditorUI {
 		return this._editableElementsMap;
 	}
 
+	/**
+	 * Returns viewport offsets object
+	 * { top: Number, right: Number, bottom: Number, left: Number }
+	 * Only top property is currently supported
+	 *
+	 * @private
+	 * @return {Object}
+	 */
+	get _viewportOffset() {
```

Comment:
OTOH, if you need to expose it for test purposes, which we sometimes do to make them simpler, it can be a method. Just avoid such complex getters. `_readViewportOffsetFromConfig()` would work better for me.

Reply:
After some thought I realised what actually this method do is to normalize viewportOffset config. Shall I maybe extract it to a separate (even public) function similar to [normalizeToolbarConfig()](https://github.com/ckeditor/ckeditor5/blob/master/packages/ckeditor5-ui/src/toolbar/normalizetoolbarconfig.js)?

What do you think @Reinmar ?

Reply:
It makes sense to extract it if it's going to be reused somewhere else. If not, keep it local cause KISS.

Reply:
Left it as a private method called `_readViewportOffsetFromConfig`